pytest_watch/watcher.py

```diff
@@ -92,8 +94,8 @@ def run(self, summary=None):
             print(STYLE_NORMAL + msg + Fore.RESET + Style.NORMAL)
         if self.beforerun:
             os.system(self.beforerun)
-        exit_code = subprocess.call(['py.test'] + self.args,
-                                    shell=(sys.platform == 'win32'))
+        exit_code = pytest.main(self.args, plugins=[ReportCollectorPlugin()])
```

Comment:
@bendtherules In light of something like `--runner`, I'm not sure adding `ReportCollectorPlugin` is worth the additional code. Unless we can figure out a way to have ptw to blindly pass results on to the callback executable or something.

I wouldn't mind adding some examples to the Readme.